app/src/main/java/ch/epfl/skysync/models/calendar/AvailabilityStatus.kt

```diff
@@ -1,8 +1,16 @@
 package ch.epfl.skysync.models.calendar
 
 /**
- * Represent three statuses: "OK" for available, "MAYBE" for pending confirmation, and "NO" for not
- * available
+ * Enum class representing the availability status of a person.
+ *
+ * The status can be one of the following:
+ * - OK: The person is available.
+ * - MAYBE: The person's availability is pending confirmation.
+ * - NO: The person is not available.
+ * - UNDEFINED: The availability status is not defined.
+ * - ASSIGNED: The person is assigned to a task.
```

Comment:
I would prefer: The person is assigned to a flight.

Reply:
Yeah, I have changed that